com.ownrwallet.desktop.desktop

```
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Type=Application
+Name=OWNR Wallet
+Icon=com.ownrwallet.desktop
+Exec=/app/bin/ownrwallet %U
```

```suggestion
Exec=ownrwallet %U
```

Also the desktop file is missing things like Catergories